compiler/rustc_target/src/spec/hexagon_unknown_linux_musl.rs

```diff
@@ -9,7 +9,11 @@ pub fn target() -> Target {
 
     base.crt_static_default = false;
     base.has_rpath = true;
-    base.linker_flavor = LinkerFlavor::Unix(Cc::Yes);
+    base.linker_flavor = LinkerFlavor::Gnu(Cc::No, Lld::Yes);
+    base.dynamic_linking = true;
```

Comment:
This is already the default for linux targets.

Reply:
I will delete this.